sdk/storage/AzureStorageBlob/Source/Operation/ResumableOperation.swift

```diff
@@ -39,7 +39,7 @@ internal class ResumableOperation: Operation {
         return internalState
     }
 
-    public weak var transfer: TransferImpl?
```

Comment:
A `ResumableOperation` (which I'm considering renaming TransferOperation) must have a reference to the transfer that spawned it, meaning it should not be a weak reference.